### PR TITLE
Scto_link problem

### PR DESCRIPTION
**Pull request by kelseylarson**
Solves problems with creation of scto_link variable when the check doesn't flag any problems. Adds tests for this situation to the test files.

### CODE REVIEW

File: ado/ipacheckreplace.ado [hunk 1]
```diff
@@ -0,0 +1,47 @@
+
```
Comment (boyercb) [inline ado/ipacheckreplace.ado]: @kelseylarson - Did you  commit this file by mistake? I know Caton said you were working on modifying `readreplace` to deal with `select_multiple` variables. If it's ready we should still move it to a separate commit/PR to improve version control.

File: ado/ipacheck.pkg [hunk 2]
```diff
@@ -29,4 +29,4 @@ f ipacheckdates.ado
 f ipacheckoutliers.ado 
 f ipacheckenum.ado
 f ipacheckresearch.ado
-f ipachecksummary.ado
\ No newline at end of file
+f ipatracksummary.ado
```
Comment (boyercb) [inline ado/ipacheck.pkg]: @kelseylarson - Good catch! This is an issue I noticed this morning but I'd like to address it in a separate commit if possible.

File: ado/ipacheckreplace.ado (hunk 1 above)
Comment (kelseylarson) [inline ado/ipacheckreplace.ado]: Ah, that was a mistake! I have it on another branch, so deleting it won't be a problem.
